src/main/archetype/ui.content/src/main/content/jcr_root/content/__appId__/language-masters/en/.content.xml

```diff
@@ -26,22 +28,66 @@
                 sling:resourceType="${appId}/components/container">
                 <title
                     jcr:primaryType="nt:unstructured"
-                    jcr:title="Welcome to My Site!"
                     sling:resourceType="${appId}/components/title"/>
                 <container
                     jcr:primaryType="nt:unstructured"
                     sling:resourceType="${appId}/components/container"
                     layout="responsiveGrid">
-                    <text
+                    <teaser
```

Comment:
Do we want to have these elements added too ?

Reply:
@CezCz i'm not sure what you mean? The only spa (react, angular) components that are implemented is the text component and layout container AFAICT...